android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -348,4 +360,79 @@ open class Node<V : RibView>(
 
     override fun toString(): String =
         "Node@${hashCode()} ($identifier)"
+
+    /**
+     * Executes an action and remains on the same hierarchical level
+     *
+     * @return the current workflow element
+     */
+    protected inline fun <reified T> executeWorkflow(
```

Comment:
How about transforming them to extension functions?
This class is already overloaded (almost 400 lines) and I suspect it will grow further

Reply:
Seems like a dead end for now either due to exposing things to be public or due to different kinds of compilation errors on a language level. Common decision: not worth it.